### DISCUSSION TITLE
Factional Doctrines

### DISCUSSION THREAD

**Discussion by PurpleMossClump** [Team-QUIET/QUIET-Community-Edition#181] [Ideas]
The purpose of this discussion is as follows:

- Establish a consensus for what each of the factions' doctrines is understood to be
- Clearly define each doctrine as a foundation for design decisions moving forward
- Discuss exceptions and outliers, e.g. Experimental Units, and how much play a faction has for breaking the mold of their established doctrine

doctrine [noun]
doc·​trine [ˈdäk-trən ]
a: a principle or position or the body of principles in a branch of knowledge or system of belief 
b: a statement of fundamental government policy especially in international relations
c: a principle of law established through past decisions
--d: a military principle or set of strategies--
e: something that is taught

We will be referring to 'd' as our definition moving forward.
An example of what a 'doctrine' might be in this context:

**UEF Doctrine focuses on adaptability. They have no obvious clear strengths or easily exploitable weaknesses. 
Their units range, agility, firepower and cost are fairly average. Their sturdy, built-to-last construction gives their units and structures more HP than any other faction's equivalents.
They are the most 'traditional' military, intending to meet the enemy head-on and outlast them in a straight fight.** 

After we have a consensus on each faction's doctrine, we can then move into more granular detail, defining how those doctrines are reflected in the stats and performance of their units.

**Comment by PurpleMossClump** [comment]
As a note: That is not necessarily how I feel what the UEF doctrine should specifically be; That is just an example.

**Comment by WhiteFang3** [comment]
Here my take on all factions:

**UEF**
A faction focuses on conventional warfare. They usually employ tanks or units on tread and don't work well with amphibious operations. This should be a good starter faction for players to learn the fundamentals, the aesthetics are familiar to real world and it's function should be easy to understand: A jack of all trade, master of none. All of their units are tough, get the job done and they are not afraid of fighting in the front line, with solid multi-role units in land, air and sea. They also got good options in artillery including a dedicated shield to deflect artillery and an experimental global artillery. 

Notable Experimentals or Units: 
Fatboy: a mobile base with a slightly higher build power than a T3 factory, they are not intended on going toe-to-toe with other experimental or armies, but acting as a moving base capable of air staging, producing an army and providing fire support. An army should always be around these. Signature experimental of the UEF.  

Nova unit cannon: producing units at a staggering pace and launches them to any place in the battlefield to quickly respond to any attack or to employ shock tactics. Weakness are the unit launched are vulnerable to anti air and only launches units one at a time instead as a group at once. Works great to quickly reinforce a position from long distance, but Fatboy can fulfill that role and then some.   

Mayhem: while labeled as a brawler, it feels like it functions as a mobile weapons platform. They can get overwhelmed with swarm of chaff units or air, so they are often used to take out high value targets within the safety of an army. 

Dragon Nite or Goliath: They would be a good brawler experimental as the tip of the spear: solid dps, reasonable turn rate and fire rate capable of dealing with an army or an experimental but due to the very high price point being the 1st and 2nd most expensive land unit respectively for the UEF and having low survivability for return, they were rarely being used in the front line and functions similarly to a . Suggest discussion of rework at a later date.

Citadel Mk2: A good experimental to help secure the air and act as a staging for air units, but can get swarmed by fighters and killed maybe a bit too fast without proper support.   

Satellite facility: being the only faction that employs this yet other has a more cost-effective way of gaining intel(i.e scrying from Aeon, spy plane spam) with subpar dps, it's only saving grace that the satellite was invulnerable to damage barring Seraphim experimental PD. Now with anti nuke structure that can kill satellite, it is not a good investment for recon.  

Summary: Easy to learn, hard to master faction. Doesn't shy away from a brawl of any kind and are flexible enough for most roles.
Pro: Front line fight, all rounded, static artillery 
Con: Amphibious operation, limited mobility

**Comment by WhiteFang3** [comment]
**Cybran**
Guerilla warfare faction and quite possibly the most dangerous faction to play against. Employing mostly arachnids units and bots, they specialized in disruption and stealth. Few units in this faction can brawl in the front line, but they make up for it by utilizing stealth units and cloakers in conjunction with units delivering high volume of missiles capable of stunning units. After all, the most effective defense is when the target cannot shoot back. Cybrans live and die by intel: denying the enemy targets and pinning them in place for indirect fire while staying undetected and making decisive strikes on high value targets. A difficult faction to play, but highly rewarding.

Notable Experimental or Units:
Mermaid Counter Intel: A T2 unit with very high value. Since sonar stations don't have omni, the Mermaid can make a navy undetectable unless the enemy regularly scan the sea with spy planes. Dangerous when combined with subs or working in conjunction with air. 

Salem class: The only navy ship that can walk on land. A destroyer with long range guns and good damage makes it's dangerous against land army.   

Monkeylord: Signature experimental of the Cybran well known within the community in vanilla. Boasting high damage and durability for it's cost, amphibious and capable of stealth they can cause a lot of damage to targets in sneak attack and raids. They can function as a front line experimental, but they are best when working as a raider and in pack.

Megalith: Another signature experimental of the Cybran. Like the Fatboy, can produce units in a form of deploying an egg, but must be stationary for the egg to "hatch". Unlike the Fatboy, the Megalith can work in the frontline and is a decent combat experimental with good range, but can be swarmed. Best used with an army

Basilisk: A adaptable experimental. Can be used as a brawler or change into the potent siege mode but can wander too close to enemy. A good experimental to have.

Avalanche: Most expensive experimental in the modpack, often used as "style points". It is very tanky and it's got good dps, but the resources required to build this are better spent elsewhere.  

Summary: Guerilla warfare faction, out-range and outmaneuver enemy with emphasis on stealth and missiles. Hard faction to play as. 
Pro: Counter Intel, Disruption tactics
Con: Front line, fragile units

**Comment by WhiteFang3** [comment]
**Aeon**
A Specialist faction, sleek and shiny with an emphasis on hover units. Most of their unit perform a single function, making them inflexible and having to rely on combined arms. They can use conventional tactics and have units that can take a beating, but success will require an understanding of the units used and the tactics employed. Act as a middle ground between UEF and Cybran. Due to a reliance on hover units, they can work in tandem with their navy units and can even hold their own against other navy units. A unique faction mechanic of this is a Miasma, an aoe that damages units staying within it. Aeon require focus to play, a clear plan on what to do and may come out as too rigid, but when it works it is hard to stop. 

Notable Experimental or Unit:
Predator: The only sniper capable of firing on the move. Kite units very well with it's range and a staple T2 unit to include in an army.

Motha: High performing Experimental PD, very good range and damage. Can be swarmed, but can "creep" building Mothas to great effects.

Torrent: A missile boat excel at killing high value targets. Designed to overwhelms missile defense. 

Eliash: Once a very lethal front line unit, it been tone down. Still works as a good front line unit, but best employed in packs and with support.

Galactic/Universal Colossus: Signature unit of Aeon. Specialist unit to destroy non-experimental units with staggering efficiency. 

Angel: Mobile guardian experimental. One of the few unit that can act as a mobile anti nuke. It's selling point would be it's anti nuke able to cover both navy and land, got a good shield coverage. 

Inquisitor: A siege experimental. teleport ordinance in a 3x3 square with good aoe. Hard countered by anti-teleport that redirect shots back to Inquisitor. Good in it's role.

Absolution: Artillery experimental tank. Capable of stunning and is hover, good to work with navy. Is inaccurate, best used with multiple.

Genesis: Artillery experimental hovercraft. Basically a hover Torrent with Miasma. Provide good accurate indirect fire support.

King Kraptor: Experimental bot. Has large Miasma field that deals heavy damage and is capable of deflect projectiles. A good front line experimental.

CZAR: Experimental flying saucer. Aircraft carrier capable of producing air units, have a powerful main gun requiring flying directly above a unit and can drop depth charges. On death, drops on the surface to deal massive damage to any under it . Best used to clear static position

Summary: A faction best used with an overall strategy in mind. Highly specialized.  
Pro: Combined arms, strong specialist units, amphibious operations
Con: Rigid, lack of multi-role units

**Comment by WhiteFang3** [comment]
**Seraphim**
Disclaimer: the units and building of these factions are a mouthful to write and speak.   
A fanatical faction, strange but sleek and employ alien technology different from the other factions. They employ expendable, suicidal units intended on dealing the most damage before and after being destroyed and mostly favors close range engagements. Most notably, the Lamba field that can destroy or even redirect the projectile back to sender ignoring defenses and protector units that buffs nearby units and draw fire. Due to the volatile nature of the units employed, it is often preferable for Seraphim to lead the attack to avoid collateral damage to nearby friendly units. Most of their navy can submerge to avoid fire from the surface. 

Notable Experimental or Unit:
(I'm calling the lighting ball on death a Death rattle)
Ythotha: Experimental assault bot. Has Death rattle

Yenthotha: experimental assault tank. It hovers and has a beam that increase in damage the longer it been fired. Has drones to assist in combat and has Death rattle.

T4 Nuke Launcher: Experimental nuke launcher that requires 2 anti nuke to neutralize. It's designed to overwhelm anti nuke and deals increased damage over other nukes. 

Lamba assault bot: Employs Lamba tech to destroy and reflect projectile.

Lobster Bot: An experimental melee unit. Is amphibious and is suppose to specialized in melee but doesn't have the necessarily tools nor survivability to do so.

Summary: Fanatic purifiers laying waste at any cost. CQC faction
Pro:Lamba field, cheap and expendable units
Con: Volatile, close range

**Comment by PurpleMossClump** [comment]
UEF

Traditional warfare; Rapid Deployment; Hammer/Anvil Tactics;

Favors fighting face-to-face, using overwhelming force to beat down their opponents. 
Units are built sturdier than their contemporaries, generally leading to more favorable engagements in a vacuum, but sacrifice agility for the extra armor.
Generally unwilling to make compromises in designs, leading to little specialization. Units that are specialized see more of a premium in their cost, or other downsides.

Favors rapid deployment utilizing high-capacity, well-armed and armored transport dropships to quickly relocate their armies that are otherwise quite mediocre in their agility. 

Fortifications are built to last. Shields make no compromises, leading to all around decent statistics and energy upkeep. 
This makes their entrenched positions overall decent in every respect.

**Comment by AmericanSniper** [comment]
UEF T1 LAND: Simple
UEF T2 LAND: Simple with shields
UEF T3 LAND: Simple with fast titans-relative unit, and slow-percival type units
UEF EXPERIMENTAL: Not simple Land and Air battleship, and simple fast Bloodasp, and simple slow Mayhem.
UEF T1 AIR: Simple with napalm bombers
UEF T2 AIR: Simple with napalm bombers & good raptors
UEF T3 AIR: Simple
UEF T1 NAVY: Simple
UEF T2 NAVY: Short range destroyer with long range cruiser, not simple torpedo boat, simple submarine & shield boat.
UEF T3 NAVY: Simple battlecruiser first, simple battleship later
